### ISSUE TITLE
[cordova] Resize observer causes ERROR Internal navigation rejected - <allow-navigation> not set for url='about:blank'

### ISSUE THREAD

**Issue by cgslivre** [quasarframework/quasar#5901]
**Describe the bug**
Cordova App on IOS (Iphone/Ipad) emit this error:
ERROR Internal navigation rejected - &lt;allow-navigation&gt; not set for url='about:blank'

**Codepen/jsFiddle/Codesandbox (required)**
Simple Cordova APP

**To Reproduce**
1 - Create new app
2 - Add cordova mode
3 - Build IOS
4 - On Xcode view output console

Possible problem origin:
quasar/src/components/resize-observer/QResizeObserver.js

**QResizeObserver** fragment
![image](https://user-images.githubusercontent.com/2656420/70963779-497f7980-2068-11ea-9bf3-6ba8d5c52087.png)

**Output** xcode console
![image](https://user-images.githubusercontent.com/2656420/70963825-6caa2900-2068-11ea-90ce-686a76b94630.png)

**config.xml** Attempt to solve with allow-intent and allow-navigation without success
![image](https://user-images.githubusercontent.com/2656420/70963864-8c415180-2068-11ea-848d-73c9af4611d4.png)

**Comment by pdanpdan** [comment]
According to this source: https://stackoverflow.com/questions/36234957/internal-navigation-rejected-allow-navigation-not-set-in-cordova-on-ios
it should be
```
<access origin="about:*" />
<allow-navigation href="about:" />
```
or
```
<access origin="about:*" />
<allow-navigation href="about:*" />
```

**Comment by rstoenescu** [comment]
What are your cordova cli and iOS versions?

**Comment by rstoenescu** [comment]
Hi,

Thanks for reporting!
Fix available in "@quasar/app" v1.4.2.

You can add `<allow-navigation href="about:*" />` in your /src-cordova/config.xml (at root level in the xml file) before the release of next q/app too -- cause that's what the next q/app will do :)
